N/nlohmann_json/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "nlohmann_json"
+version = v"3.11.3"
+
+sources = [
+   GitSource("https://github.com/nlohmann/json/", "9cca280a4d0ccf0c08f47a99aa71d1b0e52f8d03")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+
+mkdir build
+cd build
+
+CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+            -DCMAKE_FIND_ROOT_PATH='${prefix}/lib/mpich;${prefix}' \
```

Comment:
Why `${prefix}/lib/mpich`?

Reply:
An artifact left over from the recipe I copied from, my bad